core/src/main/scala/scalaz/FreeAp.scala

```diff
@@ -22,7 +22,7 @@ sealed abstract class FreeAp[F[_],A] {
   def para[B](pure: A => B, ap: λ[α => (F[α], FreeAp[F, α => A])] ~> λ[α => B]): B =
     this match {
       case Pure(x) => pure(x)
-      case x@Ap() => ap(x.v() -> x.k())
+      case x@Ap() => ap(x.v(), x.k())
```

Comment:
this uses Scala's very dodgy auto-tupling, to do do this properly there needs to be a second set of parens around the tuple:

    ap((x.v(), x.k()))

The original version explicitly passed a tuple in, the new version is ambiguous whether it is passing a tuple parameter or two arguments.

https://issues.scala-lang.org/browse/SI-3583